static/styles/typography.less

```diff
@@ -98,6 +98,7 @@ strong, b {
 }
 em,cite{
 	font-style:italic;
+	font-weight:600;
```

Comment:
Good point. How's this?
![image](https://cloud.githubusercontent.com/assets/6282922/25355234/c09fc628-28ea-11e7-8557-f96758c1a21e.png)